tools/build_config/Jenkinsfile

```diff
@@ -487,6 +508,11 @@ pipeline {
         ]
       )
 
+      archiveArtifacts(
```

Comment:
now both this and the `cppcheck.xml/mlint.json` archiveArtifacts calls are `allowEmptyArchive` do they need to be separate, or is there clarity in keeping them the analysis and test outputs separate?

Reply:
Windows doesn't run analysis, needs those to be able to be empty.
`do-not-build/test` need to allow `testing.xml` to be empty. 

Separation is because they're calculated on different stages and are different things.